### ISSUE TITLE
Translations change v8 > v9

### ISSUE THREAD

**Issue by pedrobaeza** [OCA/OpenUpgrade#690]
I have just detected that the comments to find the translation of fields texts have changed.

In version 8, you have following translation comment for the field label (the `string` attribute)
```
#: field:model.name.with.dots,field_name:0
```

However, in version 9, this is the equivalent comment for translating the field label:
```
#: model:ir.model.fields,field_description:module_name.field_xml_id
```
Being `field_xml_id` generally composed of:  `field_` + `model_name_with_underscores` + `_` + `field_name` (I'm not sure if `model_name_with_underscores` can vary if the `_table_name` attribute is modified in the model).

Are you aware of this, @jbeficent, @hbrunn, @StefanRijnhart ? Have you handled this in any way?

**Comment by StefanRijnhart** [comment]
Currently not handled, I'm afraid. All custom field label translations will be reset by the looks of it.

**Comment by pedrobaeza** [comment]
As there's no easy solution, I close this although it's a pity to lose custom translations from v8 to v9.

**Comment by StefanRijnhart** [comment]
@pedrobaeza I don't agree with closing issues just because we don't have a solution yet. Do you mind if I reopen it?

**Comment by StefanRijnhart** [comment]
What is the biggest problem in this? Is it the performance of a couple of thousands of sql queries?

**Comment by pedrobaeza** [comment]
No, the problem is to determine how to transform old "syntax" to the new one. Check first comment for a bigger explanation.

There's an additional problem that I can expand with fields with `translate=html_translate`.

**Comment by StefanRijnhart** [comment]
Doesn't the following query give you the source and target value for the transformation?

```
SELECT 'field:'||imf.model||','||imf.name||':0',
    'model:ir.model.fields,field_description:'||imd.module||'.'||imd.name 
    FROM ir_model_data imd
    JOIN ir_model_fields imf ON imf.id = imd.res_id
    WHERE imd.model = 'ir.model.fields';
```

**Comment by pedrobaeza** [comment]
But you have to change existing ones to new ones, so that doesn't serve as is. I'm also doubting about the part of model name with underscores.

**Comment by StefanRijnhart** [comment]
I misunderstood. Your comment features the description from the translation file, not from the translation table. But I still don't understand the problem :-(

**Comment by StefanRijnhart** [comment]
Do you want to change the translations in the database or in the translation files of custom modules?

**Comment by pedrobaeza** [comment]
Existing translations in the database. I will write further about this later.

**Comment by StefanRijnhart** [comment]
Closed with #1241
